lib/shared/widgets/common_icons.dart

```diff
@@ -104,48 +104,56 @@ Widget day4TimerIconRed({double? width, double? height}) => SvgPicture.asset(
     );
 Widget primaryCircleWithText({
   String? text = '',
-  double? circleSize = 90,
+  double circleSize = 90,
   Color? circleColor,
   Color? iconColor,
 }) =>
-    Container(
-      width: circleSize,
-      height: circleSize,
-      decoration: BoxDecoration(
-        color: circleColor ?? FamilyAppTheme.primary95,
-        shape: BoxShape.circle,
-      ),
-      child: Center(
-        child: Text(
-          text!,
-          style: TextStyle(
-            color: iconColor ?? FamilyAppTheme.primary20,
-            fontSize: text.length < 3 ? 40 : 35,
-            fontWeight: FontWeight.bold,
+    Padding(
+      // 14 is what the design has when exporting svgs
+      padding: const EdgeInsets.all(14),
+      child: Container(
+        width: circleSize - 28,
+        height: circleSize - 28,
+        decoration: BoxDecoration(
+          color: circleColor ?? FamilyAppTheme.primary95,
+          shape: BoxShape.circle,
+        ),
+        child: Center(
+          child: Text(
+            text!,
+            style: TextStyle(
+              color: iconColor ?? FamilyAppTheme.primary20,
+              fontSize: text.length < 3 ? 40 : 35,
+              fontWeight: FontWeight.bold,
+            ),
           ),
         ),
       ),
     );
 
 Widget primaryCircleWithIcon({
   IconData? iconData,
-  double? circleSize = 90,
-  double? iconSize = 40,
+  double circleSize = 90,
+  double iconSize = 40,
   Color? circleColor,
   Color? iconColor,
 }) =>
-    Container(
-      width: circleSize,
-      height: circleSize,
-      decoration: BoxDecoration(
-        color: circleColor ?? FamilyAppTheme.primary95,
-        shape: BoxShape.circle,
-      ),
-      child: Center(
-        child: Icon(
-          iconData,
-          color: iconColor ?? FamilyAppTheme.primary20,
-          size: iconSize,
+    Padding(
+      // 14 is what the design has when exporting svgs
+      padding: const EdgeInsets.all(14),
+      child: Container(
+        width: circleSize - 28,
```

Comment:
I would not do that here. I would change the defaultParam to circleSize = 68. Because now if you pass a size you will go cuckoo, like "WHY IS IT NOT THE SIZE I PASSED".